### PR TITLE
Transition to sox

### PR DESCRIPTION
**Pull request by privatezero**
Using sox to grab the signal allows basic tracking through the CLI of peaked audio as well as (in combination with buffering) appears to solve dropped sample problem on slower mac

### CODE REVIEW

**Comment by retokromer** [comment]
Is there any reason I miss for not using `$(...)` instead of ``` `...` ```?

**Comment by privatezero** [comment]
That is just a legacy of using Carsten Bleum's rather old Pashua calling function- I think it can certainly be updated!

**Comment by retokromer** [comment]
Thank you! I guess it makes the code more readable, therefore easier maintainable in the future.

**Comment by retokromer** [comment]
See question in code, otherwise LGTM

File: audiorecorder [hunk 1]
```diff
@@ -229,17 +216,17 @@ FILTER_CHAIN="asplit=6[out1][a][b][c][d][e],\
...
+    rec -r "${SAMPLE_RATE_NUMERIC}" -b 32 -L -e signed-integer --buffer 131072 -p | ffmpeg -i - -f wav -c:a pcm_s16le -ar 44100 - |\
     ffplay -window_title "Audio Recorder" -f lavfi \
     "amovie='pipe\:0',${FILTER_CHAIN}"
     stty sane
     exit
 fi
 
 _get_ITEM_ID
-
+rm -r PIPE2REC
```
Comment (retokromer) [inline audiorecorder]: Is there always a file to remove? If not, I would suggest to trap the error message and redirect `>/dev/null`

Comment (privatezero) [reply audiorecorder]: Good point! Maybe I will move that to the end of the script so that it cleans up after recording instead of before.

Comment (retokromer) [reply audiorecorder]: Good solution!

**Pull request merged**